src/Modules/Roles/roleCtrl.js

```diff
@@ -0,0 +1,58 @@
+const CustomError = require("../../Errors/CustomError");
+const successResponse = require("../../Utils/apiResponse");
+const asyncHandler = require("../../Utils/asyncHandler");
+const RoleService = require("./roleService");
+
+const roleCtrl = {
+    create: asyncHandler(async (req, res, next) => {
+      const roleData = req.body;
+
+    const savedRole = await RoleService.create(roleData);
+
+    return successResponse({
+      res: res,
+      data: savedRole,
+      msg: "user created Successfully",
+    });
+    }),
+  
+    getAll: asyncHandler(async (req, res, next) => {
+    
+     const{savedData,totalCount}=await RoleService.getAll()
```

Comment:
pass roleDTO object in RoleService.getAll() method so we can access request payload in future.